### DISCUSSION TITLE
回不去了 ... VSCode

### DISCUSSION THREAD

**Discussion by Linya-IronMan** [nshen/learn-neovim-lua#66] [General]
neovim 用了有一段时间了，虽然还没有完全替代日常的生产，但是已经回不去了 .... 
才用了 hjkl 多久，就完全不想再去动方向键，总觉得去动方向键好麻烦好麻烦 ... 
总是习惯性 jj，回退到 normal 模式 ... 

不过，打造自己的vim工作流还有很长的一段路要走 ......  好烦，

**Comment by allworldg** [comment]
现代ide都有vim插件,可以用来过渡
